xsc_test.go

```diff
@@ -29,3 +32,51 @@ func TestReportError(t *testing.T) {
 	errorToReport := errors.New("THIS IS NOT A REAL ERROR! This Error is posted as part of TestReportError test")
 	assert.NoError(t, utils.ReportError(serverDetails, errorToReport, "cli"))
 }
+
+func TestXscAuditNpmJson(t *testing.T) {
+	// Make sure the audit request will work with xsc and not xray
+	err := os.Setenv("JFROG_CLI_REPORT_USAGE", "")
+	assert.NoError(t, err)
+	defer func() {
+		err = os.Setenv("JFROG_CLI_REPORT_USAGE", "false")
+		assert.NoError(t, err)
+	}()
+	output := testAuditNpm(t, string(format.Json))
+	securityTestUtils.VerifyJsonScanResults(t, output, 1, 0, 1)
+}
```

Comment:
```suggestion
func initXscTest(t *testing.T) func() {
    // Make sure the audit request will work with xsc and not xray
	assert.NoError(t, os.Setenv(coreutils.ReportUsage, ""))
    return func() {
		assert.NoError(t, os.Setenv(coreutils.ReportUsage, "false"))
	}
}

func TestXscAuditNpmJson(t *testing.T) {
    restoreFunc := initXscTest(t)
    defer restoreFunc()
	output := testAuditNpm(t, string(format.Json))
	securityTestUtils.VerifyJsonScanResults(t, output, 1, 0, 1)
}
```
* Since we will need this for every xsc tests I suggest to create an init func for it. (replace in all tests in this file).
* Use the global variable for the option and not hard coded value